### ISSUE TITLE
Coder extension installing but login command not found

### ISSUE THREAD

**Issue by mjasten32** [coder/vscode-coder#87]
When I either manually install Coder Remote or from the web UI Dekstop button, I do not see the CLI download.

If I'm in the extension, the left pane is blank and if I click the login into Coder button, it says it is not found.

I have tried uninstalling, rebooting, and reinstalling the extension and have witnessed the same experience.

Any suggestions? I'm in an M1 Macbook, with plenty of memory 16GB. I used the Arm64 version of VS Code.

<img width="1024" alt="image" src="https://user-images.githubusercontent.com/106293630/234023324-6853bd80-500c-4e48-a85a-5d449b257c23.png">

<img width="288" alt="image" src="https://user-images.githubusercontent.com/106293630/234023446-3f0aac8e-afff-46fc-a77f-5a25e087b4d5.png">

**Comment by leonhausdorf** [comment]
I got the same error but with the MacBook Pro 2020 M1.

I also tried to install the extension into the Insiders version from VSCode and it happened to work normally at first but after a workspace restart/timeout it got "broken" again. Deleting the most known cache and reinstalling VSCode does not fix it. 

It works well on other systems like Windows. The Apple Silicon platform seems to be the only one with this problem.

I will do further investigation.
